tests/utils/LinearAlg2DTest.cpp

```diff
@@ -136,57 +133,63 @@ TEST_P(GetAngleTest, GetAngle)
+                  GetAngleParameters(Point(-100, 0), Point(0, 0), Point(-100, -1), 360), // Almost straight back but the other way around.
+                  GetAngleParameters(Point(-100, 0), Point(0, 0), Point(-100, 0), 0) // Completely straight back.
+                  ));
 
 TEST(GetAngleTest, GetAngleLeftAABTest)
 {
-    LinearAlg2D::getAngleLeft(Point(0, 0), Point(0, 0), Point(100, 0)); //Any output is allowed. Just don't crash!
+    LinearAlg2D::getAngleLeft(Point(0, 0), Point(0, 0), Point(100, 0)); // Any output is allowed. Just don't crash!
 }
 
 TEST(GetAngleTest, GetAngleLeftABBTest)
 {
-    LinearAlg2D::getAngleLeft(Point(0, 0), Point(100, 0), Point(100, 100)); //Any output is allowed. Just don't crash!
+    LinearAlg2D::getAngleLeft(Point(0, 0), Point(100, 0), Point(100, 100)); // Any output is allowed. Just don't crash!
 }
 
 TEST(GetAngleTest, GetAngleLeftAAATest)
 {
-    LinearAlg2D::getAngleLeft(Point(0, 0), Point(0, 0), Point(0, 0)); //Any output is allowed. Just don't crash!
+    LinearAlg2D::getAngleLeft(Point(0, 0), Point(0, 0), Point(0, 0)); // Any output is allowed. Just don't crash!
 }
 
 TEST(PointIsLeftOfLineTest, LeftOfLine)
 {
     constexpr short actual = 1;
-    const Point p(0, 10);  //    ^
-    const Point a(10, 0);  //  . |
+    const Point p(0, 10); //    ^
+    const Point a(10, 0); //  . |
     const Point b(10, 20); //    |
 
     const coord_t supposed = LinearAlg2D::pointIsLeftOfLine(p, a, b);
     ASSERT_TRUE(actual * supposed > 0 || (actual == 0 && supposed == 0))
-        << "Point " << p << " was computed as lying " << ((supposed == 0) ? "on" : ((supposed < 0) ? "left" : "right")) << " the line from " << a << " to " << b << ", instead of " << ((actual == 0) ? "on" : ((actual < 0) ? "left" : "right"));
+      << "Point " << p << " was computed as lying " << ((supposed == 0) ? "on" : ((supposed < 0) ? "left" : "right")) << " the line from "
+      << a << " to " << b << ", instead of " << ((actual == 0) ? "on" : ((actual < 0) ? "left" : "right"));
 }
 
 TEST(PointIsLeftOfLineTest, Sharp)
 {
     constexpr short actual = -1;
     const Point p(3896, 3975); // ^
     const Point a(1599, 3975); //  \    .
-    const Point b(200, 3996);  //   \                                      .
+    const Point b(200, 3996); //   \                                      .
 
     const coord_t supposed = LinearAlg2D::pointIsLeftOfLine(p, a, b);
     ASSERT_TRUE(actual * supposed > 0 || (actual == 0 && supposed == 0))
-        << "Point " << p << " was computed as lying " << ((supposed == 0)? "on" : ((supposed < 0)? "left" : "right")) << " the line from " << a << " to " << b << ", instead of " << ((actual == 0)? "on" : ((actual < 0)? "left" : "right"));
+      << "Point " << p << " was computed as lying " << ((supposed == 0) ? "on" : ((supposed < 0) ? "left" : "right")) << " the line from "
+      << a << " to " << b << ", instead of " << ((actual == 0) ? "on" : ((actual < 0) ? "left" : "right"));
```

Comment:
Indent with 4 spaces.

I also think this line could be broken up into more logical pieces than this.